### ISSUE TITLE
Name of window title bar is not updated after jumping to a different file via a local link 

### ISSUE THREAD

**Issue by BoolPurist** [Inlyne-Project/inlyne#360]
## What is the problem

When I click on a link pointing to a local file, then the name of the title bar stays the same. It has still the name of the file which was opened at the start.

## How to reproduce it

Folder setup
```txt
root.md
some_folder/other_file.md
```

Content of the root.md
```md
# Root

[other](./some_folder/other_file.md)
```
Content of the file "other_file.md".
```md
# Other file
```

Go into such a folder setup and then execute the following command

```txt
inlyne root.md
```

I see the following 

![root_window](https://github.com/user-attachments/assets/cfa232c4-f02e-47ac-b729-3487f35706cf)

When click on the "other" in blue as a local link.

Then I see the following 

![Other_file](https://github.com/user-attachments/assets/5e5c4a78-2f9c-4e8b-a921-f8cc5f07c357)

As can be seen here, the title bar still says "root.md". That is not right in my view.

### Log output after clicking onto the local link "other"

```txt
2024-09-14T01:05:03.043501Z  INFO inlyne::file_watcher: Updating file watcher path: some_folder/other_file.md
``` 

## Expected behavior

The title bar should show relative path in respect to the initially opened file. In this concrete example, that would be "some_folder/other_file.md".

**Comment by BoolPurist** [comment]
I am willing to fix this by myself if that is okay.
If I can do it, should I do it after the release v0.5 ?

**Comment by CosmicHorrorDev** [comment]
Id be more than happy if you want to give fixing it a go. No need to wait or anything, it just may be a bit before the change is released

**Comment by CosmicHorrorDev** [comment]
Closing this as already addressed by #274
